core/roboconf-dm/src/test/java/net/roboconf/dm/internal/autonomic/DmMessageProcessorForAutonomicTest.java

```diff
@@ -61,17 +63,22 @@
 	private TestApplication app;
 	private DmMessageProcessor processor;
 	private Manager manager;
+	private MessagingClientFactoryRegistry registry = new MessagingClientFactoryRegistry();
 
 
 	@Before
 	public void resetManager() throws Exception {
+		this.registry.addMessagingClientFactory(new TestClientFactory());
 		File dir = this.folder.newFolder();
 
 		this.manager = new Manager();
 		this.manager.setTargetResolver( new TestTargetResolver());
-		this.manager.setMessagingFactoryType( MessagingConstants.FACTORY_TEST );
+		this.manager.setMessagingType(MessagingConstants.FACTORY_TEST);
 		this.manager.setConfigurationDirectoryLocation( dir.getAbsolutePath());
 		this.manager.start();
+		// Reconfigure with the messaging client factory registry set.
```

Comment:
When you add comments this way, I think adding a break line before makes the code more readable. :)
You copied and pasted this here and there.